srcpkgs/python3-pyspotify/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pyspotify'
+pkgname=python3-pyspotify
+version=2.1.3
+revision=1
+wrksrc="pyspotify-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-setuptools python3-devel libspotify-devel"
+depends="python3-setuptools libspotify python3-cffi"
```

Comment:
libspotify does not need to be listed here, it is automatically detected:

```
=> python3-pyspotify-2.1.3_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libspotify.so.12 <-> libspotify>=12.1.51_1
```

Reply:
o.O Fixed !
Is it a new python3_module pre-pkg hook?
Can we just ignore runtimes deps if they are declared in setup.py?

Reply:
It's the runtime dependency hook that detects which shared libraries the generated binaries link against and generate the dependencies based on that. I only works for compiled code that links against shared libraries so it won't detect python dependencies or anything in setup.py.